obelisk/cpp/zoo/include/position_setpoint_controller.h

```diff
@@ -28,6 +29,12 @@ class PositionSetpointController : public obelisk::ObeliskController<obelisk_con
         std::string contents;
         params_file >> contents;
         amplitude_ = std::stof(contents);
+
+        // example of setting other configurable params in the node
+        this->declare_parameter<std::string>("test_param", "default_value");
+        std::string test_param_value_;
```

Comment:
nit: we only use the "_" at the end for member variables. but for local variables within a member function you can just use normal snake case.

Reply:
resolved